testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/oid4vc/issuance/signing/OID4VCSdJwtIssuingEndpointTest.java

```diff
@@ -0,0 +1,397 @@
+                        ),
+                authenticator,
+                new ObjectMapper(),
+                TIME_PROVIDER,
+                30,
+                true);
+    }
+
+    private ComponentExportRepresentation getIdCredentialSigningProvider() {
+        ComponentExportRepresentation componentExportRepresentation = new ComponentExportRepresentation();
+        componentExportRepresentation.setName("sd-jwt-signing_identity_credential");
+        componentExportRepresentation.setId(UUID.randomUUID().toString());
+        componentExportRepresentation.setProviderId(Format.SD_JWT_VC.toString());
+
+        componentExportRepresentation.setConfig(new MultivaluedHashMap<>(
+                Map.of(
+                        "algorithmType", List.of("ES256"),
+                        "tokenType", List.of(Format.SD_JWT_VC.toString()),
+                        "issuerDid", List.of(TEST_DID.toString()),
+                        "hashAlgorithm", List.of("sha-256"),
+                        "decoys", List.of("0"),
+                        "vct", List.of("https://credentials.example.com/identity_credential"),
+                        "vcConfigId", List.of("IdentityCredential")
+                )
+        ));
+        return componentExportRepresentation;
+    }
+
+    private ComponentExportRepresentation getTestCredentialSigningProvider() {
+        ComponentExportRepresentation componentExportRepresentation = new ComponentExportRepresentation();
+        componentExportRepresentation.setName("sd-jwt-signing_test-credential");
+        componentExportRepresentation.setId(UUID.randomUUID().toString());
+        componentExportRepresentation.setProviderId(Format.SD_JWT_VC.toString());
+
+        componentExportRepresentation.setConfig(new MultivaluedHashMap<>(
+                Map.of(
+                        "algorithmType", List.of("ES256"),
+                        "tokenType", List.of(Format.SD_JWT_VC.toString()),
+                        "issuerDid", List.of(TEST_DID.toString()),
+                        "hashAlgorithm", List.of("sha-256"),
+                        "decoys", List.of("2"),
+                        "vct", List.of("https://credentials.example.com/test-credential"),
+                        "vcConfigId", List.of("test-credential")
+                )
+        ));
+        return componentExportRepresentation;
+    }
+
+    @Override
+    protected ClientRepresentation getTestClient(String clientId) {
+        ClientRepresentation clientRepresentation = new ClientRepresentation();
+        clientRepresentation.setClientId(clientId);
+        clientRepresentation.setProtocol(OID4VCLoginProtocolFactory.PROTOCOL_ID);
+        clientRepresentation.setEnabled(true);
+        Map<String, String> testCredentialAttributes = Map.of(
+                "vc.test-credential.expiry_in_s", "1800",
+                "vc.test-credential.format", Format.SD_JWT_VC.toString(),
```

Comment:
It might be `Format.SD_JWT_VC`

Reply:
Resolved with [d8cd042](https://github.com/keycloak/keycloak/pull/30692/commits/d8cd042dc629ebb73c665be5831483de8b9c2174)